client-app/src/examples/contact/desktop/AppModel.ts

```diff
@@ -1,8 +1,6 @@
 import {XH} from '@xh/hoist/core';
-import {ContactService} from './svc/ContactService';
-import {BaseAppModel} from '../../BaseAppModel';
-
-export const PERSIST_APP = {prefKey: 'contactAppState'};
```

Comment:
As desktop AppModel is no longer the root for all of contacts, I moved this to [here](https://github.com/xh/toolbox/pull/752/files#diff-59d924fa7e5d7a8e651c4f0064a0dd29045a0743817cc572ce38000b1d9fe89bR5), which is common to both versions.